src/webhook/handlers/processImage.js

```diff
@@ -70,10 +70,17 @@ export default async function({ data = {} }, event, userId) {
 
       // choose for user
       event = {
-        type: 'postback',
+        type: 'server_choose',
```

Comment:
Question: what would happen if we just use `'postback'` here?

Reply:
No functional difference.

I use `server_choose` just to sperate two different flow : "user choose" and "server choose for user".